judge/admin/profile.py

```diff
@@ -66,6 +66,7 @@ def get_fields(self, request, obj=None):
         if request.user.has_perm('judge.totp'):
             fields = list(self.fields)
             fields.insert(fields.index('is_totp_enabled') + 1, 'totp_key')
+            fields.append('scratch_codes')
```

Comment:
Right now, this shows the `scratch_codes` field at the bottom of the profile admin page. I think scratch codes should be right below `totp_key`, so maybe something like
```
            fields.insert(fields.index('totp_key') + 1, 'scratch_codes')
```